.rhdh/scripts/install.sh

```diff
@@ -148,7 +143,7 @@ elif [[ -z $CLUSTER_ROUTER_BASE ]]; then
   exit 1
 fi
 
-# 3. change values
+# change values
 helm upgrade redhat-developer-hub -i "${CHART_URL}" --version "$CV" \
     --set global.clusterRouterBase="${CLUSTER_ROUTER_BASE}" \
     --set global.postgresql.auth.password="$PASSWORD"
```

Comment:
```suggestion
helm upgrade redhat-developer-hub -i "${CHART_URL}" --version "$CV" \
    --set global.clusterRouterBase="${CLUSTER_ROUTER_BASE}"
```

`$PASSWORD` is useless then.